packages/blueprints-integration/src/rundown.ts

```diff
@@ -292,6 +299,13 @@ export interface IBlueprintPieceInstance<TMetadata = unknown> {
 
 	piece: IBlueprintPieceDB<TMetadata>
 
+	/** The time the system started playback of this part, null if not yet played back (milliseconds since epoch) */
+	startedPlayback?: Time
```

Comment:
Is the comment "... _undefined_ if not yet played back ... " or the type `Time | null` ?

Reply:
should read as undefined. Thats what I get for moving a property and comments without reading it..